index.html

```diff
@@ -80,6 +81,10 @@
               Meetups are held on the second and fourth Monday each month. Come to code, learn, meet like-minded people, and have fun!
           </p>
         </article>
+      <a href="#carousel-example-generic" id="back-to-top-btn">
```

Comment:
I think it would be better to anchor this to the title h1 instead of the carousel, as I suspect that the carousel id will change with time.